x-pack/metricbeat/module/statsd/server/server.go

```diff
@@ -177,11 +177,11 @@ func (m *MetricSet) getEvents() []*mb.Event {
 
 		for k, v := range tagGroup.metrics {
 			ms := eventMapping(k, v, m.mappings)
-			if ms == nil || len(*ms) == 0 {
+			if ms == nil || len(ms) == 0 {
```

Comment:
```suggestion
			if len(ms) == 0 {
```

As the type is `T` instead of `*T`, just `len` is enough as `T` is `mapstr.M` here which is a `map`.

Reply:
Previously it was required as the returned type was a pointer to a map, not a map. So a nil check was important.